assets/src/apps/delivery/store/features/adaptivity/actions/triggerCheck.ts

```diff
@@ -328,8 +328,28 @@ export const triggerCheck = createAsyncThunk(
     attempt.dateEvaluated = Date.now();
 
     await dispatch(upsertActivityAttemptState({ attempt }));
+    let doesCheckResultContainsNavigationToDifferentScreen = false;
```

Comment:
Just moved this logic up here so that same code can be used for determining if check result has a wrong trap state with navigation to a different screen.